datafusion/core/src/datasource/statistics.rs

```diff
@@ -160,17 +159,6 @@ pub(crate) fn create_max_min_accs(
     (max_values, min_values)
 }
 
-fn add_row_stats(
```

Comment:
It is interesting that there is very similar code in https://github.com/apache/arrow-datafusion/blob/a892300a5a56c97b5b4ddc9aa4a421aaf412d0fe/datafusion/core/src/datasource/file_format/parquet.rs#L503-L525 that uses `add` 🤔

Reply:
It seems like it is always adding exact values, so there should be no difference on the results whether it is used `add()` or `estimated_add()`